src/wp-admin/security.php

```diff
@@ -0,0 +1,194 @@
+        <span style="color: #BA2121">&#39;My Menu Title&#39;</span>,
+        <span style="color: #BA2121">&#39;my-prodigious-plugin&#39;</span>,
+        <span style="color: #BA2121">&#39;my-output-function&#39;</span>
+    );
+} <span style="color: #008000; font-weight: bold">else</span> {
+    <span style="color: #408080; font-style: italic">// existing code</span>
+}
+</pre>
+              </div>
+              <p>You will also need to change the logic on your settings pages, but that is outside the scope of this guide. However, you should remember that the idea is to <strong>move</strong> security-related settings, <strong>not</strong> to <em>duplicate</em> them.</p>
+            </div>
+            <h2><span><?php _e( 'Security Plugins' ); ?></span></h2>
+            <div class="inside">
+              <p>If your plugin has nothing but security-related settings it may be more useful to provide a summary of the current settings, with links to where they can be changed.</p>
+            </div>
+            <h2><span><?php _e( 'Plugin Action Links'); ?></span></h2>
+            <div class="inside">
+              <p>Many plugins add links to the plugin action links. [As above wrt rephrasing] You can declutter that area for your plugin by using a standard dashicon; for example, the "Settings" link:</p>
+              <!-- HTML generated using hilite.me -->
+              <div style="background: #f8f8f8; overflow:auto;width:auto;border:solid gray;border-width:.1em .1em .1em .8em;padding:.2em .6em;">
+<pre style="margin: 0; line-height: 125%"><span style="color: #008000">array_unshift</span>(
+    <span style="color: #19177C">$links</span>,
+    <span style="color: #008000">sprintf</span>(
+        <span style="color: #BA2121">&#39;&lt;a href=&quot;%s?page=my-prodigious-plugin&quot; title=&quot;%s&quot;&gt;%s&lt;/a&gt;&#39;</span>,
+        admin_url( <span style="color: #BA2121">&#39;admin.php&#39;</span> ),
+        __( <span style="color: #BA2121">&#39;Settings&#39;</span> ),
+        <span style="color: #008000">function_exists</span>( <span style="color: #BA2121">&#39;\add_security_page&#39;</span> )
+            <span style="color: #666666">?</span> <span style="color: #BA2121">&#39;&lt;span class=&quot;dashicon dashicons-admin-generic&quot;&gt;&lt;/span&gt;&#39;</span>
+            <span style="color: #666666">:</span> __( <span style="color: #BA2121">&#39;Settings&#39;</span> )
+    )
+);
+</pre>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div id="postbox-container-1" class="postbox-container">
+        <div class="meta-box-sortables">
+          <div class="postbox">
+            <h2><span><?php _e( 'Support and Documentation' ); ?></span></h2>
+            <div class="inside">
+              <p>
+                <ul>
+                  <li><a href="https://forums.classicpress.net/c/support/security-page" target="_blank">Security Page forum</a></li>
```

Comment:
Should we use `link.classicpress.net` here too?

Reply:
Yes, that's a good plan - `/support/security-page`?

Reply:
Done, https://link.classicpress.net/support/security-page